templates/taxi/car_list.html

```diff
@@ -7,6 +8,12 @@ <h1>
       Create
```

Comment:
Consider adding a more descriptive label for the 'Create' button to clarify what is being created, such as 'Create Car'.